### PR TITLE
updated logger.py file

### PR DESCRIPTION
**Pull request by haripriya9647**
## Description
added inline docs for methods and class
## Relevant Issue
Issue #269  the issue here

## Affected Dependencies
none
## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/NimbleEdge/EnvisEdge/labels)
- [x] My changes are covered by tests

### CODE REVIEW

File: fedrec/utilities/logger.py [hunk 1]
```diff
@@ -4,12 +4,31 @@
 
 
 class BaseLogger(ABC):
+    """
+    This logger class performs the logging messages and
```
Comment (divyanshugit) [inline fedrec/utilities/logger.py:8]: It will be good if we can re-write it as:
```suggestion
    The `BaseLogger` performs the logging messages and
```

File: fedrec/utilities/logger.py [hunk 2]
```diff
@@ -43,6 +62,9 @@ def add_graph(*args, **kwargs):
     from torch.utils.tensorboard import SummaryWriter
 
     class TBLogger(SummaryWriter, BaseLogger):
+        """
+        Helps in visualizing the data through the tensorboard class.
```
Comment (divyanshugit) [inline fedrec/utilities/logger.py:66]: Please elaborate, You can explain how BaseLogger is being used over here and a more detailed use case of it.

**Review by divyanshugit — CHANGES_REQUESTED** [review]

File: fedrec/utilities/logger.py [hunk 3]
```diff
@@ -52,6 +108,35 @@ def log(self, *args, **kwargs):
...
+               #import summarywriter
+            >>> from torch.utils.tensorboard import SummaryWriter
+               # Create summary writer with auto generated folder name
+            >>> writer = SummaryWriter()
+            >>> r = 5
+            >>> for i in range(100):
+            >>> writer.add_scalars('run_14h', {'xsinx':i*np.sin(i/r),
+                                    'xcosx':i*np.cos(i/r),
+                                    'tanx': np.tan(i/r)}, i)
+            >>> writer.close()
```
Comment (divyanshugit) [inline fedrec/utilities/logger.py:137]: Move the example inside the inline documentation fo class

**Review by divyanshugit — CHANGES_REQUESTED** [review]

File: fedrec/utilities/logger.py [hunk 4]
```diff
@@ -9,35 +9,26 @@ class BaseLogger(ABC):
...
     The logging module in python is a way to store information
-    about the script and track of events that occur. Similarly
-    here,logging.info() helps in printing the aggregated time cost.
+    about the script and track events that occur. Similarly
+    here, logging.info() helps in printing the aggregated time cost.
 
     Visualization activities are being performed by Tensorboard by
     importing SummaryWriter.Tensorboard is a suite of web applications
-    for inspecting and understanding model runs and graphs.It supports
+    for inspecting and understanding model runs and graphs. It supports
```
Comment (ramesht007) [inline fedrec/utilities/logger.py:17]: Instead of model runs you can write `model metrics`.

**Review by ramesht007 — CHANGES_REQUESTED** [review]

**Comment by haripriya9647** [comment]
@ramesht007 I have added commit
